config/default.js

```diff
@@ -36,9 +38,21 @@ module.exports = {
   OPTIMIST_HOST: process.env.OPTIMIST_HOST || 'optimist',
   OPTIMIST_PORT: process.env.OPTIMIST_PORT || 80,
   clientBaseUrl: `http://${process.env.CLIENT_HOST}:${process.env.CLIENT_PORT}`,
-  optimistBaseUrl: `http://${process.env.OPTIMIST_HOST}:${process.env.OPTIMIST_HTTP_PORT}`,
-  optimistWsUrl: `ws://${process.env.OPTIMIST_HOST}:${process.env.OPTIMIST_WS_PORT}`,
-  web3WsUrl: `ws://${process.env.BLOCKCHAIN_WS_HOST}:${process.env.BLOCKCHAIN_PORT}`,
+  // Define Urls and Url format (http/ws vs https/wss, with vs without port) depending on whether a DOMAIN_NAME has been defined.
+  // In production and staging environements, we require https/wss and no port, as traffic will be routed to the correct service
+  // given a URL.
+  optimistBaseUrl:
```

Comment:
Can we define this in the `ENVIRONMENTS` variables below where it already exists?

Reply:
`ENVIRONMENTS` is only used in internal testing. We should not touch that.